src/components/shared/Entity/ValidationErrorSummary.tsx

```diff
@@ -37,15 +43,21 @@ function ValidationErrorSummary() {
         });
     }
 
-    if (filteredSpecErrors.length > 0) {
+    if (isEmpty(endpointSchema)) {
+        filteredErrorsList.push({
+            title: intl.formatMessage({
+                id: 'foo.endpointConfig.endpointConfigMissing',
+            }),
+        });
+    } else if (filteredSpecErrors.length > 0) {
```

Comment:
Checking if the endpoint schema is empty and informing the user to select a connector.